chat_app_be/chat_app_be/Controllers/ConversationController.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using chat_app_be.Dtos;
+using chat_app_be.Services.Interfaces;
+
+namespace chat_app_be.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ConversationController : ControllerBase
+    {
+        private readonly IConversationService _conversationService;
+        private readonly ILogger<ConversationController> _logger;
+
+        public ConversationController(IConversationService conversationService, ILogger<ConversationController> logger)
+        {
+            _conversationService = conversationService;
+            _logger = logger;
+        }
+
+        [HttpPost("createConvo")]
```

Comment:
We can leave this route as default.
To create a conversation, it will use "/api/conversation" with the POST method. Similar to get one, get many, update, and delete a conversation.